### PR TITLE
[FINAL] Use libc++ instead of libstdc++

### PR DESCRIPTION
**Pull request by tonidero**

### CODE REVIEW

File: src/uslscore/uslscore.h [hunk 1]
```diff
@@ -87,7 +87,6 @@
 #include <uslscore/USTypedPtr.h>
 #include <uslscore/USTypeID.h>
 #include <uslscore/USUnion.h>
-#include <uslscore/USUnique.h>
```
Comment (antonmorant) [inline src/uslscore/uslscore.h:90]: Never used?

**Review by antonmorant — APPROVED** [review]
Looks good to me.

File: src/uslscore/uslscore.h (hunk 1 above)
Comment (tonidero) [reply src/uslscore/uslscore.h:90]: Yes, I tried to look for references everywhere but didn't find any. Also, was the only class that was giving issues with libc++ so just decided to remove it completely

Comment (antonmorant) [reply src/uslscore/uslscore.h:90]: sg

**Pull request merged**